src/start_server.js

```diff
@@ -63,7 +64,11 @@ function startWithPort(options) {
     res.redirect(301, `/components/${polyserve.packageName}/`);
   });
 
-  app.use('/components/', polyserve);
+  if (options.cors) {
+    app.use('/components/', cors(), polyserve);
```

Comment:
`cors` is an official `express` module for adding CORS headers to some set of routes.

Is using the `express` package better than the built-in `http` module? The answer is yes and no depending on how you measure better. How are we measuring it?